indico/modules/users/client/js/UserDelete.jsx

```diff
@@ -0,0 +1,212 @@
+import ReactDOM from 'react-dom';
+import {Button, Message, Modal, Icon, List, Popup} from 'semantic-ui-react';
+
+import {Translate, Param} from 'indico/react/i18n';
+import {indicoAxios, handleAxiosError} from 'indico/utils/axios';
+
+function UserDeleteDialogBody({firstName, lastName, disabled, inProgress, onDelete, onClose}) {
+  const [countdown, setCountdown] = useState(10);
+  const [isButtonDisabled, setButtonDisabled] = useState(true);
+
+  useEffect(() => {
+    setCountdown(10);
+    setButtonDisabled(true);
+    const timer = setInterval(() => {
+      setCountdown(prevCountdown => {
+        if (prevCountdown <= 1) {
+          clearInterval(timer);
+          setButtonDisabled(false);
+          return 0;
+        }
+        return prevCountdown - 1;
+      });
+    }, 1000);
+    return () => clearInterval(timer);
+  }, []);
+
+  return (
+    <>
+      <Modal.Header>
+        <Translate>
+          Delete <Param name="first_name" value={firstName} />{' '}
+          <Param name="last_name" value={lastName} />?
+        </Translate>
+      </Modal.Header>
+      <Modal.Content>
+        <Message negative icon>
+          <Icon name="warning sign" />
+          <Message.Content>
+            <Message.Header>
+              <Translate>This action is irreversible</Translate>
+            </Message.Header>
+            <Translate>Deleted user accounts cannot be restored.</Translate>
+          </Message.Content>
+        </Message>
+        <Translate as="p">Once deleted, the following will happen:</Translate>
+        <List style={{marginTop: 0}}>
+          <List.Item>
+            <List.Icon name="minus circle" />
+            <List.Content>
+              <Translate>
+                <Param name="first_name" value={firstName} /> will no longer be able to access
+                Indico.
+              </Translate>
+            </List.Content>
+          </List.Item>
+          <List.Item>
+            <List.Icon name="times circle outline" />
+            <List.Content>
+              <Translate>
+                <Param name="first_name" value={firstName} /> will be removed from all events,
+                contributions, and other areas of Indico.
+              </Translate>
+            </List.Content>
+          </List.Item>
+          <List.Item>
+            <List.Icon name="trash alternate outline" />
+            <List.Content>
+              <Translate>
+                Where it is not possible to delete <Param name="first_name" value={firstName} />,
+                they will be anonymized and all personal data associated with the user will be
+                removed from Indico.
+              </Translate>
+            </List.Content>
+          </List.Item>
+        </List>
+        <p>
+          <Translate as="strong">
+            Are you sure you want to delete "<Param name="first_name" value={firstName} />{' '}
+            <Param name="last_name" value={lastName} />
+            "?
```

Comment:
Are quotes necessary here?

Reply:
Probably `wrapper={<strong />}` would be nicer

Reply:
There is already a wrapper strong around the entire text.

Reply:
I'll remove the quotes and put the wrapper only around the name.